docs/pages/blog/mui-x-v7.md

```diff
@@ -0,0 +1,383 @@
+
+- [Data Grid](#data-grid)
+  - [Column resizing on the Community plan](#column-resizing-on-the-community-plan)
+  - [Sticky headers and improved scrolling performance](#sticky-headers-and-improved-scrolling-performance)
+  - [Improved columns panel design](#improved-columns-panel-design)
+  - [Date Object support in filter model](#date-object-support-in-filter-model)
+  - [Smaller bundle size](#smaller-bundle-size)
+  - [New stable features](#new-stable-features)
+- [Tree View](#tree-view)
+  - [Rich Tree View](#rich-tree-view)
+  - [New customization hook: `useTreeItem2`](#new-customization-hook-usetreeitem2)
+- [Charts](#charts)
+  - [Gauge charts](#gauge-charts)
+  - [Click event handler](#click-event-handler)
+  - [Built-in Grid](#built-in-grid)
+  - [Reference Line](#reference-line)
+- [Date and Time Pickers](#date-and-time-pickers)
+  - [Improved accessibility on Pickers Text field](#improved-accessibility-on-pickers-text-field)
+  - [Support for date-fns v3](#support-for-date-fns-v3)
+  - [Date Time Range Picker](#date-time-range-picker)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+- [What's next](#whats-next)
+- [Long-term support](#long-term-support)
+- [Migration guide](#migration-guide)
+- [How to get involved](#how-to-get-involved)
+
+## Data Grid
+
+### Column resizing on the Community plan
+
+As MUI X commercial offerings continue to expand, we're pleased to enhance the community plan by integrating features previously exclusive to commercial users.
+
+[Column resizing](https://mui.com/x/react-data-grid/column-dimensions/#resizing) and [auto-sizing](https://mui.com/x/react-data-grid/column-dimensions/#autosizing), which were once Pro features, are now available to all users at no cost.
+This transition reflects our commitment to providing an exceptional user experience across the board, and aims to make the free components stand out more in the open-source space.
+
+### Sticky headers and improved scrolling performance
+
+The Data Grid now offers a more responsive experience with smoother scrolling and enhanced screen reader support, thanks to a refactor of column headers and pinning implementations.
+
+On its first announcement, we showcased some of these improvements, but during the beta phase, we have further polished the scrolling performance for an even more significantly improved user experience.
+<figure style="padding:0; margin:0">
+  <video preload="metadata" autoplay muted loop playsinline width="690" height="417" controls>
+    <source src="/static/blog/mui-x-v7/sticky-headers-and-vertical-scrolling.mp4" type="video/mp4">
+  </video>
+  <figcaption style="color: #888; font-size: 0.9em; text-align: center;">A before and after showcase, first with horizontal and then with vertical scrolling.</figcaption>
+</figure>
+
+:::warning
+The scrolling experience is far better, but the solution has a downside: **column headers can no longer be transparent** because content scrolls behind them.
```

Comment:
```suggestion
The scrolling experience is far better, but the solution has a downside: ** the column headers can no longer be transparent** because content scrolls behind them.
```